### PR TITLE
Labels added to contact form and placeholder font changed

### PR DESCRIPTION
**Pull request by tanyapowell**
@EChesters @lili2311 
### Contact Form
- Labels added to Contact Form
- Padding between some labels and input fields corrected
- Drop Shadow removed from submit button (See [Issue #22](https://github.com/womenhackfornonprofits/whfnp/issues/22))
  image
### Volunteers Sign Up form
- Drop Shadow removed from submit button (See [Issue #22](https://github.com/womenhackfornonprofits/whfnp/issues/22))
- Padding between some labels and input fields corrected
  image

### CODE REVIEW

**Comment by lili2311** [comment]
could you add normalize as a partial and import it?

**Comment by lili2311** [comment]
If we have labels should the placeholder be actual examples of what we are after?

**Comment by tanyapowell** [comment]
I think the placeholders need changing on both the contact form and volunteer form, but would like @EChesters UX opinion on it

**Comment by tanyapowell** [comment]
Also, [could we add normalize as a dependency](https://github.com/JohnAlbin/normalize-scss)?

**Comment by EChesters** [comment]
Placeholders need to be an example of the format and data that we're expecting. Labels look good now. 
On the volunteers sign up why are we just asking for surname and not full name or both(first/last)? Also not entirely convinced why we should be asking for LinkedIn, but re-looking at that form which fields are required?

Placeholder examples would be:
Full name: 
Jon Snow

Email:
jon.snow@example.com

Twitter:
@IKnowNothing

**Comment by tanyapowell** [comment]
LinkedIn is a client requirement. Will modify the placeholders to be examples 😊

**Comment by EChesters** [comment]
Are all the fields required? Because if so we will need to way of telling the users. (and not by using a star)
Why do clients need LinkedIn profiles? 

👍

**Comment by tanyapowell** [comment]
@lili2311 are all the fields required? If yes, I'll create another PR for them

### Placeholder changes

##### Contact us

image

##### Volunteer sign up

image

**Comment by lili2311** [comment]
As long as the normalize dependency doesn't re-download when someone new wants to clone project, ideally we don't want the look of the site to change just because there is a newer version of normalize. 

The forms already have `required` set some basic required fields, this has good overall support, this prompts the user to fill the field in if they have not done so already (try filling the form out on site): http://caniuse.com/#search=required
Contact: all required but Phone

Volunteer & Non-Profits Contact: we could do with confirming what else but name and email is a required field.

**Comment by lili2311** [comment]
There is also this form: https://s3-eu-west-1.amazonaws.com/whfnp.co.uk/non-profits-contact.html, will you be amending this too in this PR?

**Comment by tanyapowell** [comment]
_Normalize_: you can specify the version of the dependency in the package.json

Non-profits currently looks like the below image, so will just modify the placeholder text: 
image

File: src/css/screen.css [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /*! normalize.css v2.1.3 | MIT License | git.io/normalize */
-@import url(https://fonts.googleapis.com/css?family=Lobster);
+@import url(https://fonts.googleapis.com/css?family=Lato:400,300,700);
```
Comment (lili2311) [inline src/css/screen.css:2]: weirdly 2 imports

**Comment by lili2311** [comment]
Okies add the non profits contact form here and then :+1:

File: src/css/screen.css (hunk 1 above)
Comment (tanyapowell) [inline src/css/screen.css:2]: no idea why that is. I'm delete it when I commit the non-profit contact form change :shaved_ice:

**Comment by tanyapowell** [comment]
[Issue #30](https://github.com/womenhackfornonprofits/whfnp/issues/30) also fixed

image

**Comment by lili2311** [comment]
Please also post mobile screenshots?

**Comment by tanyapowell** [comment]
Non-Profits
image

Contact Us 
image

Volunteer
image

**Comment by lili2311** [comment]
:+1: after merge conflict is resolved.

**Pull request merged**